source/Octostache.Tests/ConditionalsFixture.cs

```diff
@@ -67,6 +67,22 @@ public void ConditionalUnlessIsSupported()
             result.Should().Be("result");
         }
 
+        [Theory]
+        [InlineData("#{if MyVar}True#{/if}", "")]
+        [InlineData("#{unless MyVar}False#{/unless}", "False")]
+        public void UnknownVariablesAreTreatedAsFalsy(string template, string expected)
+        {
+            var result = Evaluate(template, new Dictionary<string, string>());
+            result.Should().Be(expected);
+        }
+
+        [Fact]
+        public void UnknownVariablesOnBothSidesAreTreatedAsEqual()
+        {
+            var result = Evaluate("#{if Unknown1 == Unknown2}Equal#{/if}", new Dictionary<string, string>());
+            result.Should().Be("Equal");
+        }
```

Comment:
I've checked these cases against version `3.1.0` and validated that they represent the previous behaviour, which regressed in version `3.2.0` 👍